lang/src/org/partiql/lang/eval/builtins/ExtractExprFunction.kt

```diff
@@ -66,11 +67,32 @@ internal class ExtractExprFunction(valueFactory: ExprValueFactory) : NullPropaga
         }.toDouble()
     }
 
+    private fun Time.extractedValue(datePart: DatePart) : Double {
+        return when (datePart) {
+            DatePart.HOUR -> localTime.hour.toDouble()
+            DatePart.MINUTE -> localTime.minute.toDouble()
+            DatePart.SECOND -> secondsWithFractionalPart.toDouble()
+            DatePart.TIMEZONE_HOUR -> timezoneHour?.toDouble() ?: errNoContext(
```

Comment:
Should this be called `DatePart` when the fields are _time_ related?

Reply:
Will rename it to `DateTimePart`.

Reply:
Would it be okay to punt this as renaming it to `DateTimePart` also encircles renaming a lot of places in the code including variable names, error messages, function names, etc.

Reply:
Add an issue.

Reply:
Added.